azavea/pfb-network-connectivity
# Feature/awf/django score model

## Overview

This is the other followup to #222 in which it was decided to move the score metadata into the Django application so it lives in a single place. This PR adds the model and initial data fixture to load the initial set of analysis score metadata.

### Notes

Did not add additional checks or score metadata, since its not critically important. We can do so once we have a better idea of how this will be displayed to the application user.

## Testing Instructions

Run `./scripts/django-manage loaddata analysis-score-metadata` and ensure it runs successfully.

Closes #224
